src/Common/tests/System/AssertExtensions.cs

```diff
@@ -169,5 +193,31 @@ private static string AddOptionalUserMessage(string message, string userMessage)
             if (actual.CompareTo(lessThanOrEqualTo) > 0)
                 throw new XunitException(AddOptionalUserMessage($"Expected: {actual} to be less than or equal to {lessThanOrEqualTo}", userMessage));
         }
+
+        /// <summary>
+        /// Validate that a given value is greater than or equal to another value.
+        /// </summary>
+        /// <param name="actual">The value that should be greater than or equal to <paramref name="greaterThanOrEqualTo"/></param>
+        /// <param name="greaterThanOrEqualTo">The value that <paramref name="actual"/> should be greater than or equal to.</param>
+        public static void GreaterThanOrEqualTo<T>(T actual, T greaterThanOrEqualTo, string userMessage = null) where T : IComparable
```

Comment:
This could be written in just "one" line

```C#
+        public static void GreaterThanOrEqualTo<T>(T actual, T greaterThanOrEqualTo, string userMessage = null) where T : IComparable
 +        {
 +             if ((actual == null && greaterThanOrEqualTo != null) || actual.CompareTo(greaterThanOrEqualTo) < 0)
 +                throw new XunitException(AddOptionalUserMessage($@"Expected: {actual ?? ""<null>""} to be greater than or equal to {greaterThanOrEqualTo ?? ""<null>""}", userMessage));
 +        }